website/docs/reference/global-configs/resource-type.md

```diff
@@ -4,8 +4,18 @@ id: "resource-type"
 sidebar: "resource type"
 ---
 
+<VersionBlock lastVersion="1.8">
+
 The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types from the `dbt build`, `dbt clone`, and `dbt list` commands.
```

Comment:
non blocker but thought it might be helpful for 1.8 and lower users to know that newer versions/versionless supports dbt test
```suggestion
The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types from the `dbt build`, `dbt clone`, and `dbt list` commands. Starting in Versionless and dbt v1.9, these flags are also supported in the `dbt test` command.
```